lib/api/core/hotelClerk.js

```diff
@@ -225,11 +226,166 @@ module.exports = function HotelClerk (kuzzle) {
 
     return deferred.promise;
   };
+
+  /**
+   * Remove rooms for a given collection
+   * If rooms attribute is not provided, all rooms for the collection are removed
+   * @param {RequestObject} requestObject
+   * @returns {Promise} resolve a response object. May have PartialError if rooms are provided and don't match collection
+   */
+  this.removeAllRooms = function (requestObject) {
```

Comment:
As this method may remove a selection of rooms, shouldn't it be called ``removeRooms`` instead?